CriteoPublisherSdk/Tests/UnitTests/CR_BidManagerTests.m

```diff
@@ -406,7 +407,7 @@ - (void)testAddCriteoBidToRequestCallHeaderBidding {
 - (void)testAddCriteoBidToRequestWhenKillSwitchIsEngagedShouldNotEnrichRequest {
```

Comment:
I'm not sure that this scenario should be kept. If killSwitch is on, you won't even get a bid. Maybe should you either remove the test, either improve it to test getBid+enrichAdObject.

Reply:
Will remove with the logic tested.

Reply:
Done